### PR TITLE
Convert fluent getters to Kotlin getters (Fix #15)

### LINKED ISSUE #15: Convert getter method to Kotlin getters in Config
Convert these to Kotlin getters as it's more usual to do it like that in Kotlin:
https://github.com/Votebot/bot/blob/55c775278cf527868cdd5cdbf0cf7aa9c86bf33b/shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt#L29-L31

### PR DESCRIPTION
**Pull request by DRSchlaubi**
- Deprecate fluent getters
- Migrate to Kotlin getters
- Upgrade deprecated calls

### CODE REVIEW

**Comment by ForYaSee** [comment]
Delete it

File: shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt
Comment (yk-sgr) [inline shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt]: Delete

File: shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt
Comment (yk-sgr) [inline shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt]: Delete

File: shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt
Comment (yk-sgr) [inline shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt]: Delete

**Review by yk-sgr — CHANGES_REQUESTED** [review]

File: shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt
Comment (leonkappes) [inline shardManager/src/main/kotlin/space/votebot/shardmanager/config/Config.kt]: Remove entirely

**Review by leonkappes — CHANGES_REQUESTED** [review]

**Review by yk-sgr — APPROVED** [review]

**Review by leonkappes — APPROVED** [review]

**Comment by leonkappes** [comment]
Do not merge until further notice

**Comment by leonkappes** [comment]
Can't you read?!?

**Comment by DRSchlaubi** [comment]
> Do not merge until further notice

Couldn't you have said that a bit earlier? The message literally showed up in the same second I hit the merge button

**Pull request merged**
